src/reducer/videoReducer.js

```diff
@@ -0,0 +1,79 @@
+import {
+  ADD_TO_PLAYLIST,
+  CREATE_PLAYLIST,
+  DATA_FROM_SERVER,
+  ADD_TO_HISTORY,
+  LIKED_VIDEOS,
+  SEARCH_VIDEO,
+  WATCH_LATER
+} from '../constants/reducerConstants'
+import { checkItem } from '../utils/checkItem'
+
+const checkVideoExists = (playlist, videoId) =>
```

Comment:
I have a suggestion here, if "checkVideoExists" function is used multiple times, we can move it to utils folder

Reply:
Yes, thanks I have created utils folder for the same
Missed to refactor the code